resources/resource_ids.spec

```diff
@@ -182,6 +182,10 @@
     "META": {"sizes": {"includes": [250]}},
     "includes": [59270]
   },
+  "<(SHARED_INTERMEDIATE_DIR)/brave/web-ui-commands/commands.grd": {
+    "META": {"sizes": {"includes": [250]}},
```

Comment:
I also don't know much about this ids spec file but this seems break existing ones.
It looks like above playlist grd has 250 includes and starts at `59270` and below one(ledger_bridge) is next (`59520` == `59270 + 250`).

Reply:
no, this file does not map IDs directly, it only used as a guide. The actual ID mapping is dynamic and does not overlap IDs from different groups (it's a build step that will just fail).

Reply:
Updated to have a distinct id at the end of the file anyway